pkg/ccl/backupccl/restore_job.go

```diff
@@ -1237,7 +1238,9 @@ func createImportingDescriptors(
 		dbsByID[databases[i].GetID()] = databases[i]
 	}
 
-	if !details.PrepareCompleted {
+	if !details.PrepareCompleted && !details.DryRun {
```

Comment:
by skipping this branch, I skip switching the status to JobRunning. I need to fix this.